app/shared/api-services/poll/answer-factory.tests.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+describe('Answer Factory', function () {
+  var $httpBackend,
+      Answer,
+      answer,
+      mockPayload,
+      response,
+      scope,
+      pollId;
+
+  beforeEach(module('apiServices.answer.factory'));
+
+  beforeEach(inject(function(_$httpBackend_, _Answer_) {
+    Answer = _Answer_;
+    $httpBackend = _$httpBackend_;
+
+    mockPayload = {
+      id: 203,
+      answer_text: 'texty text',
+      poll: 123456
+    };
+
+  }));
+
+  describe('postAnswer()', function () {
+    it('makes a post request', function () {
+      answer = {id: 2, notOnSodahead: true, answer_text: 'wingapo'};
+      pollId = 12;
+      Answer.postAnswer(answer, pollId);
+      $httpBackend.expectPOST('/cms/api/v1/answer/').respond(201);
+      $httpBackend.flush();
+    });
+
+    it('returns answer payload', function() {
+      answer = {id: 2, notOnSodahead: true, answer_text: 'wingapo'};
+      pollId = 12;
+      Answer.postAnswer(answer, pollId).then(function(res) {
+        response = res;
+      });
+      $httpBackend.expectPOST('/cms/api/v1/answer/').respond(201, mockPayload);
```

Comment:
I think we're still not making any assertion on what we're posting to the endpoint, which I think is important